node-fs-writefile/note-to-self.js

```diff
@@ -0,0 +1,13 @@
+import { writeFile } from 'node:fs/promises';
+
+async function randomNote() {
+  const note = process.argv[2];
+  const write = await writeFile('note.txt', note + '\n');
+  console.log(write);
```

Comment:
`writeFile`'s promise doesn't resolve to anything useful, so the logging here could be removed.